test/viewmodels/helpers/MovementPaginationHelperSpec.scala

```diff
@@ -31,12 +31,12 @@ class MovementPaginationHelperSpec extends SpecBase with MovementListFixtures {
   object Helper extends MovementPaginationHelper()
 
   def createPageItem(index: Int): PaginationItem = PaginationItem(
-    href = onPageLoad(testErn, MovementListSearchOptions(Some(ARC), Some("ARC123456"), ArcAscending, index, 10)).url,
+    href = onPageLoad(testErn, MovementListSearchOptions(Some(ARC), Some("ARC123456"), ArcAscending, None, None, None, None, None, None, None, None, None, index, 10)).url,
```

Comment:
Probably wotth having a default `MovementListSearchOptions` instance at the top of the test, and just changing the `index` and `maxRows` fields where relevant